### DISCUSSION TITLE
zabbix configure database

### DISCUSSION THREAD

**Discussion by gjkrisa** [tteck/Proxmox#3770] [Help Needed] [answered]
trying to get past the configure database i used default everything as setup and defaults in database added port 5454 user: Admin password zabbix then added cert location as /root
i tried alot of things even nano the cert and tried copying those credentials but look encypted or something.
error cannot connect to database
thank you for your time and for making this script
![1](https://github.com/user-attachments/assets/06cec3fa-1d5e-40bc-ae67-758fdd2f6819)
![2](https://github.com/user-attachments/assets/d2d65c70-7111-4681-9043-a18cf0844ffc)

**Comment by bbreton09** [comment, marked as answer]
Answer is here : https://github.com/tteck/Proxmox/discussions/3458#discussioncomment-10301010
